lib/src/models/submission_impl.dart

```diff
@@ -191,6 +191,8 @@ class Submission extends SubmissionRef
   ///
   /// Returns `true` if the submission is upvoted, `false` if it is downvoted,
   /// and `null` otherwise.
+  /// depreciated use [vote]
```

Comment:
Replace this line with the following (including "empty" line) to be consistent with the comment in `comment_impl.dart`:

```dart
///
/// This property is deprecated. Please use `vote` instead.
@deprecated
```